src/App.jsx

```diff
@@ -1,7 +1,20 @@
 import './App.css';
+import { ThemeProvider } from '@emotion/react';
+import { theme } from './Theme/index';
+import { Typography } from '@mui/material';
+import { Route, BrowserRouter, Routes } from 'react-router-dom';
 
 function App() {
-  return <div>App *2</div>;
+  return (
+    <ThemeProvider theme={theme}>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<div>home page</div>} />
```

Comment:
I know this is initial setup.
but please consider starting it with clean code.
for example here the element should be a component in a separate file that has this dev.